javascript/exercises/credit_card_validator.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
```

Comment:
This looks like it should go in `jsweb/` instead of `javascript/`.

Is this a solution to an exercise problem?